src/Template/Pages/Dashboard/index.twig

```diff
@@ -40,25 +49,21 @@
                     { '_name': 'modules:list', 'object_type': 'objects' },
                     { 'title': __('All objects'), 'class': 'dashboard-item has-background-module-objects icon-th-large-1', } )|raw }}
 
-                {# <a href="#" class="dashboard-item has-background-black icon-list-1" disabled>{{ __('All categories') }}</a> #}
+                <a href="#" class="dashboard-item has-background-module-categories icon-list-1">{{ __('All categories') }}</a>
```

Comment:
`All categories` and `All tags` are not available yet - better keep both anchors commented

Reply:
Note: we have `Categories` under `Data modeling` and it's ok to handle them there.

If we want to have a standalone `All categories` module to be used by non-admin users we will probably clone it or (better) prepare two possible views.

Reply:
Ok, I agree on a specific `Categories` view for non-admin, and keeping both anchors commented for now.